net-proxy/haproxy/files/1.6.9-MINOR-build-Allow-linking-to-device-atlas-library-fi.patch

```diff
@@ -0,0 +1,28 @@
+From 01dc6e622c4eceab80c9bacd2ae2c1228a6791c4 Mon Sep 17 00:00:00 2001
+From: Bertrand Jacquin <bertrand@jacquin.bzh>
+Date: Thu, 6 Oct 2016 00:32:39 +0100
+Subject: [PATCH] MINOR: build: Allow linking to device-atlas library file
+
+DeviceAtlas might be installed in a location where a user might not have
+enough permissions to write json.o and dac.o
+---
+ Makefile | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/Makefile b/Makefile
+index 0ce4b325305c..a10d2e4c1040 100644
+--- a/Makefile
++++ b/Makefile
+@@ -622,8 +622,12 @@ endif
+ DEVICEATLAS_SRC =
+ DEVICEATLAS_INC = $(DEVICEATLAS_SRC)
+ DEVICEATLAS_LIB = $(DEVICEATLAS_SRC)
++ifeq ($(DEVICEATLAS_LIB),)
```

Comment:
@bjacquin reping for this part. Also is this still needed for 1.6.10 / 1.7 releases?